attackers/client.py

```diff
@@ -0,0 +1,50 @@
+import socket
+import response_handlers
+from create_update_request import update_request
+from common.const import *
+from common import protocol
+
+
+def connect_to_server(address: ADDR) -> socket.socket:
+    my_socket = socket.socket()
+    my_socket.connect(address)
+    return my_socket
+
+
+def create_request() -> FULL_REQUEST:

```

Comment:
create give the feeling this functio is a factory.  
A factory gets all the raw materials and constructs them.  
This function does that but it also interacts with the client.  
This should be 2 different functions.